src/lib/db/index.ts

```diff
@@ -84,4 +84,8 @@ export default class Database {
       })
       .where({ id: localId })
   }
+
+  getCreationID = async (capacityId: UUID) => {
```

Comment:
rather than getting a creation ID, this is getting a creation transaction? Also the ID argument is a `creationId` or `transactionId` rather than ` capacityId`